javascript-prep/src/css/detail.css

```diff
@@ -0,0 +1,13 @@
+.detail-container{
+    padding: 2em;
+}
+
+.detail-container .qa-container .accordion .card{
```

Comment:
DOUBT -- NOT A COMMENT
Does this work like this?
As we haven't added classes for accordion or card? Does React treat them as classes?

Reply:
Yes, so the accordion has its own classes and I have provided CSS on the inbuilt classes. It works!

Reply:
It's like normal Bootstrap

Reply:
No, I meant, why does .accordion work? shouldn't it be only Accordion?

as in, for `<div class="a">`, we would either say
.a {_style_} or div {_style_}

Reply:
Bootstrap itself assign classes, so the Component Accordion has been assigned a class named `accordion` .  Similarly, Card component has assigned it a class name `card`. And hence we can use these classes for our css, because at the end of the day whole code is HTML in the DOM.